fakenet/listeners/FTPListener.py

```diff
@@ -182,6 +192,12 @@ def open(self, filename, mode):
             # Calculate absolute path to a fake file
             filename = os.path.join(os.path.dirname(filename), EXT_FILE_RESPONSE.get(file_extension.lower(), 'FakeNetMini.exe'))
 
+        # Collect NBIs
+        nbi = {"command": "RETR", "filename": filename, "mode": mode}
```

Comment:
this line should come before line 193. We need the filename before we do os.path.join()